src/terrain/terrain.cpp

```diff
@@ -396,7 +396,7 @@ Terrain::get_path_type(
     else
         type = horizontal_direction;
 
-    bool open;
+    bool open = false;
```

Comment:
Rename please, `open(2)` is a POSIX function: https://man7.org/linux/man-pages/man2/open.2.html